tests/unittests/test_handler/test_schema.py

```diff
@@ -447,4 +448,27 @@ def test_all_integration_test_cloud_config_schema(self):
         if errors:
             raise AssertionError(', '.join(errors))
 
+
+class TestSchemaDocExamples:
+    @skipUnlessJsonSchema()
+    def test_schema_doc_examples(self):
+        ignored_files = [
+            'cloud-config-archive-launch-index.txt',
+            'cloud-config-archive.txt',
+        ]
+
+        examples_dir = Path(
+            __file__).parent.parent.parent.parent / 'doc' / 'examples'
+        # Make sure our path shenanigans found an actual directory
+        assert examples_dir.is_dir()
+
+        all_text_files = [f for f in examples_dir.glob('cloud-config*.txt')
+                          if f.name not in ignored_files]
+        for text_file_path in all_text_files:
+            try:
+                validate_cloudconfig_file(str(text_file_path), get_schema())
+            except:  # noqa
+                print('Failed on {}'.format(text_file_path))
+                raise
```

Comment:
This will only show us the first failure that occurs, without performing tests across all the files. (Parametrizing this test will address this issue.)